go.mod

```diff
@@ -22,6 +22,7 @@ require (
 	k8s.io/apiextensions-apiserver v0.21.4
 	k8s.io/apimachinery v0.21.4
 	k8s.io/client-go v0.21.4
+	k8s.io/klog v1.0.0
```

Comment:
can this be replaced by klog/v2, because we switched to newer version as part of the CAPI v1a4 migration 

xref: https://github.com/kubernetes-sigs/cluster-api/blob/main/docs/book/src/developer/providers/v1alpha3-to-v1alpha4.md#klog-version